tutor/src/components/icons/add.cjsx

```diff
@@ -0,0 +1,12 @@
+React = require 'react'
+
+IconAdd = ->
+  <svg width="28px" height="28px" viewBox="0 0 28 28">
+    <g id="Symbols" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+      <g id="icon-add-class" fill="#575958">
+        <path d="M28,0 L28,28 L0,28 L0,0 L28,0 Z M0,1 L1,1 L1,27 L0,27 L28,27 L27,27 L27,1 L28,1 L0,1 Z M13.4,9 L14.6,9 L14.6,13.4 L19,13.4 L19,14.6 L14.6,14.6 L14.6,19 L13.4,19 L13.4,14.6 L9,14.6 L9,13.4 L13.4,13.4 L13.4,9 Z" id="Combined-Shape"></path>
+      </g>
+    </g>
+  </svg>
+
+module.exports = IconAdd
```

Comment:
Somethings going wrong with this and it doesn't appear to be rendering.  See screenshot at bottom